index.html

```diff
@@ -120,16 +208,26 @@ <h2>Multi-Post <br>Stories</h2>
             <li><h5>css</h5></li>
             <li><h5>javascript</h5></li>
          </ul>
-         <button type="button">See Project</button>
+         <button class="button" type="button">See Project</button>
+        </div>
         </div>
     
     </section>
+          </div>
     <section>
     
     <div class="about">
-        <h2>About <br> MySelf</h2>
+        <section id="about">
+        <section id="about001">
+        <h2>About MySelf</h2>
         <p class="intro">
-            Hello I am a software developer! I can help you build a product ,feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.
+            Hello I am a software developer!
+             I can help you build a product
+              ,feature or website Look through
+               some of my work and experience!
+                if you like what you see and 
+                have a project you need coded,
+                 don't hesitate to contact me.
```

Comment:
- [ ] I would suggest you stick to the Figma template design. follow the `font size`, `font weight`, `text alignment`, and the `social media icon alignment`.
- [ ] The directional icon in front of `Frameworks` and `Skills` is not well aligned. Kindly check the template below to make corrections.

Your design 👇 

![abt](https://user-images.githubusercontent.com/78634841/206792156-7e282fad-dd67-4818-9977-9c498817ccd2.png)

Figma design 👇 

![check](https://user-images.githubusercontent.com/78634841/206791860-417fa67e-7264-4a59-a6ca-9676bb7e6544.png)